stats/lib/request-helpers.js

```diff
@@ -12,7 +12,7 @@ export const today = () => getDayAsISOString(new Date())
  * @param {URLSearchParams} searchParams
  * @param {import('node:http').ServerResponse} res
  * @param {import('pg').Pool} pgPool
- * @param {(import('pg').Pool, import('./typings').Filter) => Promise<object[]>} fetchStatsFn
+ * @param {(import('pg').Pool, import('./typings.d.ts').DateRangeFilter) => Promise<object[]>} fetchStatsFn
```

Comment:
```suggestion
 * @param {(import('pg').Pool, import('./typings').DateRangeFilter) => Promise<object[]>} fetchStatsFn
```

Reply:
We would have to revert your change when we add proper `tsc` type checking. Because we use ESM imports in our JS code, TypeScript will require us to use ESM imports in typings too. ESM imports require full file names including the extensions, and TypeScript does the same.

See e.g. https://github.com/filecoin-station/spark-evaluate/pull/252

Reply:
Should we update the other type imports to include the extension then?

Reply:
> Should we update the other type imports to include the extension then?

Yes, eventually. So far, I have done it as part of enabling type-checking.